front50-s3/src/main/java/com/netflix/spinnaker/front50/config/S3Config.java

```diff
@@ -0,0 +1,81 @@
+package com.netflix.spinnaker.front50.config;
+
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3Client;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.netflix.awsobjectmapper.AmazonObjectMapper;
+import com.netflix.spinnaker.clouddriver.aws.bastion.BastionConfig;
+import com.netflix.spinnaker.clouddriver.aws.security.AmazonClientProvider;
+import com.netflix.spinnaker.front50.model.*;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.web.client.RestTemplate;
+import rx.schedulers.Schedulers;
+
+import java.util.concurrent.Executors;
+
+@Configuration
+@Import(BastionConfig.class)
+public class S3Config {
+  @Value("${spinnaker.s3.bucket}")
+  private String bucket;
+
+  @Value("${spinnaker.s3.rootFolder}")
+  private String rootFolder;
+
+  @Bean
+  public AmazonClientProvider amazonClientProvider() {
+    return new AmazonClientProvider();
+  }
+
+  @Bean
+  public AmazonS3 awsS3Client(AWSCredentialsProvider awsCredentialsProvider) {
+    return new AmazonS3Client(awsCredentialsProvider);
+  }
+
+  @Bean
+  @ConditionalOnMissingBean(RestTemplate.class)
+  public RestTemplate restTemplate() {
+    return new RestTemplate();
+  }
+
+  @Bean
+  public ObjectMapper amazonObjectMapper() {
+    return new AmazonObjectMapper();
+  }
+
+  @Bean
+  @ConditionalOnExpression("${spinnaker.s3.enabled:false}")
```

Comment:
should this be just `s3.enabled` since the cassandra config went from `spinnaker.cassandra.enabled` to `cassandra.enabled`